lib/street_address/us.rb

```diff
@@ -665,7 +665,15 @@ class << self
       [^\w\x23]*    # skip non-word chars except # (eg unit)
       #{number_regexp} \W*
       (?:#{fraction_regexp}\W*)?
+
+      (
+      # special case for Connecticut (CT)
+      (?=.*#{street_type_regexp}.*CT.*)
```

Comment:
Should this not be `street_regexp` on this line?  Otherwise, wouldn't it match on something like "Stunning Ct." because there is a "St" in the street name?

Reply:
so street_type_regex looks somethings like this \b#{street_type}\b.   where street_type is all the different street type abbreviations (st, dr, blvd, rd, ect).  So stunning ct will be parsed correctly in the example.  The only time this code will not work is if there is a multi-word street name and one of those words is also a street_type.  The example I put in the code is 123 Mill Station Ct.  In this example, "Ct" (court) should be the street type and "Mill Station".  But it won't work that way.  It will instead parse "Station" as the street type, "Mill" as the street name, and "Ct" as the City.  I'm not sure if there is a way around this however.